Dockerfile

```diff
@@ -1,26 +1,26 @@
-FROM alpine:3.7
+FROM python:3.8.2-slim
 LABEL maintainer="smartlab-dev@mpt.mp.br"
 
+ENV PYTHONPATH /app:/usr/lib/python3.8/site-packages
```

Comment:
I think you can move this near to the end, if you need to replace it, you won't need to rebuild all stages again.

Reply:
Now that you mentioned, I have to check if it's still needed - it was there before mostly because of the package building in alpine.